back end/clustering.py

```diff
@@ -0,0 +1,359 @@
+# This file gets all the sentences from txt files with texts.
+# It then divides the text in sentences and sentences into words
+ 
+from __future__ import absolute_import, division, print_function
+import codecs
+#finds all pathnames matching a pattern, like regex
+import glob
+#log events for libraries
+import logging
+#concurrency
+import multiprocessing
+#dealing with operating system , like reading file
+import os
+#pretty print, human readable
+import pprint
+#regular expressionss
+import re
+#natural language toolkit
+import nltk
+#word 2 vec
+import gensim.models.word2vec as w2v
+from gensim import models
+#dimensionality reduction
+import sklearn.manifold
+#math
+import numpy as np
+#plotting
+import matplotlib.pyplot as plt
+#parse dataset
+import pandas as pd
+#visualization
+import seaborn as sns
+import pickle
+from scipy import spatial
+from sklearn.cluster import KMeans
+
+# Download tokenizers
+# nltk.download("punkt") # pretrained tokenizer
+# nltk.download("stopwords") # remove words like and, the, an, a, of
+
+word = "bank"
+part_of_speech = "Noun"
+language = "English"
+number_of_clusters = 2
+
+# building model 
+# Dimensionality of the resulting word vectors.
+#more dimensions mean more traiig them, but more generalized
+num_features = 300
```

Comment:
Can we define this variable locally?

Reply:
Yes, but it would be better if we left it like that, because this variable is a parameter of configuration, so it would be easier for us to change it later instead of looking for it in the function.